<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Override_Dialog</class>
 <widget class="QDialog" name="Override_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Override</string>
  </property>
  <widget class="QLabel" name="override_constraint_message_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Override Message</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="override_message_label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overridable conflict: </string>
   </property>
  </widget>
  <widget class="QLabel" name="override_message_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Would you like to override?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="override_yes_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="override_no_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
